### PR TITLE
adds bobabeam related changes

### PR DESCRIPTION
**Pull request by hyd628**
### Description/Original PRs

https://github.com/PureStake/moonbeam-docs/pull/388

### Checklist

- [x] If this requires removing old images from the `moonbeam-docs` repo, I have created a corresponding PR
- [x] If this requires adding/updating/deleting redirects, I have created a corresponding PR in the `moonbeam-mkdocs` repo
- [x] If this requires removing old variables from the `moonbeam-docs` repo, I have created a corresponding PR

### Corresponding PRs

Please link to any additional corresponding PRs here.

### CODE REVIEW

File: .snippets/text/endpoints/bobabeam.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+=== "HTTPS"
+    |  Name  |                                           RPC URL                                           |
```
Comment (eshaben) [inline .snippets/text/endpoints/bobabeam.md:2]: name can be translated

File: builders/get-started/networks/layer2/bobabase.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ description: Bobabase是Boba Layer 2部署在Moonbeam上的官方测试网。遵
 
 ![Bobabase Banner](/images/builders/get-started/networks/bobabase/bobabase-banner.png)
 
-## 概览
+## 概览 {: #introduction }
```
Comment (eshaben) [inline builders/get-started/networks/layer2/bobabase.md:10]:
```suggestion
## 概览 {: #introduction }

```

File: builders/get-started/networks/layer2/bobabeam.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    chainId: {{ networks.bobabeam.chain_id }},
+    name: 'bobabeam'
+});
+```
+
+更过关于Ethers.js和Web3.js详细的教程，请前往[Ethereum API Libraries](/builders/build/eth-api/libraries/){target=_blank}。任何以太坊钱包都可以为Bobabeam生成一个有效地址（例如：[MetaMask](https://metamask.io/){target=_blank}）。
+
+## Chain ID {: #chain-id } 
+
+Bobabeam chain ID is: `{{ networks.bobabeam.chain_id }}`, which is `{{ networks.bobabeam.hex_chain_id }}` in hex.
```
Comment (eshaben) [inline builders/get-started/networks/layer2/bobabeam.md:47]:
```suggestion
```

**Review by eshaben — CHANGES_REQUESTED** [review]
one minor thing 🚀

**Comment by hyd628** [comment]
Thanks @eshaben , fixed.

**Review by eshaben — APPROVED** [review]
LGTM ✨

**Pull request merged**